androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/AppNavigation.kt

```diff
@@ -21,7 +21,7 @@ import com.EENX15_22_17.digital_journal.android.screens.triage.healthHistory.Hea
 import com.EENX15_22_17.digital_journal.android.screens.triage.previousCare.TempPreviusCare
 
 sealed class Screen(val route: String) {
-    object Overview : Screen(route = "overview")
+    object Current : Screen(route = "current")
```

Comment:
I think this name is a bit vague (not the the previous name was much better though).

In general I think we should decide on better naming for the different views, allowing for longer ones if they help with intelligibility. This should be discussed in a group though, so it's not something that will affect this PR.